Arm/BitVec.lean

```diff
@@ -555,6 +555,11 @@ protected theorem extract_lsb_of_zeroExtend (x : BitVec n) (h : j < i) :
   simp_all
   omega
 
+@[bitvec_rules, simp]
```

Comment:
Is the `simp` tag necessary?

Reply:
It's not necessary, per-se, but it seems easier to have it for manual proofs (so that just `simp` will already pick up on it). It was also a marker that this might make for a good upstream simp-lemma